lib/ioki/model/operator/place.rb

```diff
@@ -32,6 +32,10 @@ class Place < Base
                   on:   [:read, :create, :update],
                   type: :string
 
+        attribute :depot,
+                  on:   [:read, :create, :update],
```

Comment:
sometimes it's `[:read, :create, :update]` and other times `[:create, :read, :update]` - why you not align them alphabetically? 🤷🏿 

sorry, i wanted to at least once comment on a PR 😄